src/GitLink/Linker.cs

```diff
@@ -31,14 +31,19 @@ public static class Linker
         private static readonly string RevisionPlaceholder = Uri.EscapeUriString("{revision}");
         private static readonly string PdbStrExePath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "pdbstr.exe");
         private static readonly string[] ExtensionsToIgnore = new string[] { ".g.cs" };
+        private static IReadOnlyList<string> sourceFilesList = null;
```

Comment:
Please use underscore or capital like SourceFilesList for fields. Now it looks like a local variable while it's not.